packages/icons/src/icon/Icon.tsx

```diff
@@ -15,68 +9,35 @@ export const makePrefixer =
   (...names: string[]): string =>
     [prefix, ...names].join("-");
 
-export const ICON_NAMED_SIZES = ["small", "medium", "large"] as const;
-
 const withBaseName = makePrefixer("uitkIcon");
 
-export type IconSize = typeof ICON_NAMED_SIZES[number];
-
-export interface IconProps extends HTMLAttributes<HTMLSpanElement> {
+export interface IconProps extends SVGAttributes<SVGSVGElement> {
   /**
-   * Pass props to the underlying svg element.
+   * Multiplier for the base icon size. Should be a positive integer to conform to the rest of the design system.
    */
-  SVGProps?: Partial<SVGAttributes<SVGSVGElement>>;
-  /**
-   * Size of the icon, explicit size or small/medium/large
-   */
-  size?: IconSize | number;
+  size?: number;
 }
 
-export const DEFAULT_ICON_SIZE = "small";
+export const DEFAULT_ICON_SIZE = 1;
```

Comment:
should we call this multiplier?

Reply:
NO strong preference but I think I'm happy without the suffix as it corresponds to the prop `size`